### PR TITLE
Update references from master to main

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: app/assets/javascripts/modal.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Adapted from https://github.com/projectblacklight/blacklight/blob/master/app/javascript/blacklight/modal.js
+// Adapted from https://github.com/projectblacklight/blacklight/blob/main/app/javascript/blacklight/modal.js
```
Comment (mejackreed) [inline app/assets/javascripts/modal.js:1]: Is this still true?

File: app/assets/javascripts/modal.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ Mylibrary.modal.onFailure = function(data) {
 Mylibrary.modal.receiveAjax = function (contents) {
   // does it have a data- selector for container?
   // important we don't execute script tags, we shouldn't.
-  // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/master/src/ajax/load.js?source=c#L62
+  // code modelled off of JQuery ajax.load. https://github.com/jquery/jquery/blob/main/src/ajax/load.js?source=c#L62
```
Comment (mejackreed) [inline app/assets/javascripts/modal.js:39]: Is this still true?

File: app/assets/stylesheets/modules/top_navbar.scss [hunk 3]
```diff
@@ -27,7 +27,7 @@
   }
 }
 
-// these rules are pulled from https://github.com/twbs/bootstrap/blob/master/scss/_navbar.scss
+// these rules are pulled from https://github.com/twbs/bootstrap/blob/main/scss/_navbar.scss
 // but adapted to always show the uncollapsed version, regardless of breakpoints.
```
Comment (mejackreed) [inline app/assets/stylesheets/modules/top_navbar.scss:31]: Is this still true?

File: app/assets/javascripts/modal.js (hunk 1 above)
Comment (jgreben) [reply app/assets/javascripts/modal.js]: oops.

File: app/assets/javascripts/modal.js (hunk 2 above)
Comment (jgreben) [reply app/assets/javascripts/modal.js]: oops.

File: app/assets/stylesheets/modules/top_navbar.scss (hunk 3 above)
Comment (jgreben) [reply app/assets/stylesheets/modules/top_navbar.scss:31]: oops.

**Pull request merged**